### PR TITLE
Improve flow and grammar for openbmc python installation doc

### PR DESCRIPTION
**Pull request by gurevichmark**
Commit 1 - Continue work from #5628 to improve the documentation.
Commit 2 - Formatting changes to eliminate some errors when building the docs

### CODE REVIEW

**Comment by gurevichmark** [comment]
Close/reopen to trigger Travis

File: docs/source/guides/admin-guides/manage_clusters/common/deployment/syncfile/syncfile_synclist_file.rst [hunk 1]
```diff
@@ -19,7 +19,7 @@ The path_of_src_file* should be the full path of the source file on the Manageme
 
 The path_of_dst_file* should be the full path of the destination file on target node. Please make sure path_of_dst_file* is not a existing directory on target node, otherwise, the file sync with ``updatenode -r /usr/bin/scp`` or ``xdcp -r /usr/bin/scp`` will fail.
 
-The path_of_dst_directory should be the full path of the destination directory. Please make sure *eth_of_dst_directory is not a existing file on target node, otherwise, the file sync with ``updatenode -r /usr/bin/scp`` or ``xdcp -r /usr/bin/scp`` will fail.
+The path_of_dst_directory should be the full path of the destination directory. Please make sure path_of_dst_directory is not a existing file on target node, otherwise, the file sync with ``updatenode -r /usr/bin/scp`` or ``xdcp -r /usr/bin/scp`` will fail.
```
Comment (whowutwut) [inline docs/source/guides/admin-guides/manage_clusters/common/deployment/syncfile/syncfile_synclist_file.rst]: Should we highlight these `path_of_dst_directory`  ?

File: docs/source/references/coral/cluster_mgmt/scalability/python/install/rpm.rst [hunk 2]
```diff
@@ -1,117 +1,132 @@
 Using RPM (recommended)
 =======================
 
-**Support is only for RHEL 7.5 for Power LE (Power 9)**
+.. note:: Supported only on RHEL 7.5 for Power 9 LE
```
Comment (whowutwut) [inline docs/source/references/coral/cluster_mgmt/scalability/python/install/rpm.rst]: Should be `POWER9`

File: docs/source/references/coral/cluster_mgmt/scalability/python/install/rpm.rst [hunk 3]
```diff
@@ -1,117 +1,132 @@
...
+The following repositories should be configured on your Management Node.
 
-   * RHEL 7.5 OS Repository
-   * RHEL 7.5 Extras Repository
-   * RHEL 7 EPEL Repo (https://fedoraproject.org/wiki/EPEL)
-   * Fedora28 Repo (for ``gevent``, ``greenlet``)
+   * RHEL 7.5 OS repository
+   * RHEL 7.5 Extras repository
+   * RHEL 7 EPEL repository (https://fedoraproject.org/wiki/EPEL)
+   * Fedora28 reposotory (for ``gevent`` and ``greenlet``)
```
Comment (whowutwut) [inline docs/source/references/coral/cluster_mgmt/scalability/python/install/rpm.rst]: Spelling issue here

**Comment by gurevichmark** [comment]
@whowutwut New commit with suggested improvements.

**Pull request merged**